src/Tabs/index.jsx

```diff
@@ -106,13 +99,71 @@ class Tabs extends React.Component {
   }
 }
 
-// TODO: custom validator that ensures labels and panels are the same length
 Tabs.propTypes = {
-  labels: PropTypes.oneOfType([
-    PropTypes.arrayOf(PropTypes.string),
-    PropTypes.arrayOf(PropTypes.element),
-  ]).isRequired,
-  children: PropTypes.arrayOf(PropTypes.element).isRequired,
+  tabs: PropTypes.arrayOf(PropTypes.objectOf(PropTypes.string)).isRequired,
 };
 
+// TODO: custom validator that ensures labels and panels are the same length
+
 export default Tabs;
+
+Tabs.Label = ({
+  labelId, panelId, label, index, activeTab, toggle,
+}) => {
+  const selected = index === activeTab;
```

Comment:
nit: Renaming the `selected` variable to `isSelected` might make it more clear elsewhere that this is a boolean. Same for anywhere that may use the variable `selected`.